lib/DDGC/DB/Result/InstantAnswer.pm

```diff
@@ -395,6 +395,53 @@ many_to_many 'users', 'instant_answer_users', 'user';
 has_many 'instant_answer_topics', 'DDGC::DB::Result::InstantAnswer::Topics', 'instant_answer_id';
 many_to_many 'topics', 'instant_answer_topics', 'topic';
 
+after insert => sub {
+    my ( $self ) = @_;
+    my $schema = $self->result_source->schema;
+    $schema->resultset('ActivityFeed')->create(
+        $self->ddgc->config->subscriptions->created_ia_page( {
+            meta1        => $self->id,
+            meta2        => join('', map { sprintf ':%s:', $_ }
+                $self->topics->columns([qw/ name /])->all),
```

Comment:
Yeah, it's a delimeter so you can search for '%:$topic:%' and be sure you have matched a complete topic name.

:+1: Helper sounds like a plan, thanks for heads up on pod.